web/src/app/core/services/repository.service.ts

```diff
@@ -76,4 +82,19 @@ export class RepositoryService {
 
     return ((boundary - duration) / 30) * 100; // percentage
   }
+
+  public getPointsByCount(count: number, limit: number): number {
+    let points: number;
+    switch (true) {
+      case (count >= 1 && count <= limit):
+        points = 50;
+        break;
+      case (count > limit):
+        points = 100;
+        break;
+      default:
+        points = 0;
+    }
+    return points;
```

Comment:
I prefer to have a blank line before the `return`, can we have a lint rule for this - something for the future, not a major issue for now